.vscode/launch.json

```diff
@@ -6,12 +6,12 @@
             "request": "attach",
             "smartStep": true,
             "port": 9229,
-            "sourceMaps": true,
-            "sourceMapPathOverrides": {
-                "webpack:///./~/*": "${workspaceFolder}/node_modules/*",
-                "webpack://?:*/*": "${workspaceFolder}/*",
-                "file:///app/node_modules/*": "${workspaceFolder}/node_modules/*"
-            }
+            "address": "127.0.0.1",
```

Comment:
Can we keep the old config, and maybe rename it to "Attach to Tests" or so?
It worked well for that, and I'd like to avoid the additional hour to fix it again next time.

Reply:
Ah, good to know it was used for something 👍